src/test/java/org/jenkinsci/plugins/build_token_root/BuildRootActionTest.java

```diff
@@ -133,4 +135,22 @@ private void assertCreated(Page page) throws Exception {
     // TODO test polling
     // TODO test projects in folders
 
+    @Issue("JENKINS-25637")
+    @PresetData(PresetData.DataSet.NO_ANONYMOUS_READACCESS)
+    @Test public void testCrumbBypass() throws Exception {
+        FreeStyleProject p = j.createFreeStyleProject("p");
+        setAuthToken(p);
+
+        HttpClient client = new HttpClient();
```

Comment:
I did not use `JenkinsRule.WebClient` because it automatically adds the crumb. If there's a way to avoid that, happy to change the test case to use it instead.